tests/integration/test_unique.py

```diff
@@ -31,36 +31,38 @@ def test_with_nonzero():
 
 
 @pytest.mark.parametrize("ndim", range(LEGATE_MAX_DIM + 1))
-def test_ndim(ndim):
-    shape = (4,) * ndim
+@pytest.mark.parametrize("return_index", (True, False))
+def test_ndim(ndim, return_index):
```

Comment:
I don't think that this is enough testing. I think the tricky cases around duplicates and which index is returned etc need to be checked explicitly.

Reply:
Do you think what I added in the new commit satisfies this case?

Reply:
yes it is better.